shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/profile/orders/presentation/OrderHistoryComponent.kt

```diff
@@ -111,13 +111,11 @@ open class OrderHistoryComponent(
             is OrderHistoryEvents.FetchContents -> fetchOrderHistory()
             is OrderHistoryEvents.PrimaryButtonPressed -> when(event.order.status) {
                 OrderStatus.PENDING -> commonUiState.update { it.copy(confirmAction = OrderHistoryConfirmAction.Cancellation(event.order)) }
-                OrderStatus.SHIPPED -> Unit // TODO: implement feature
-                OrderStatus.CANCELLED -> Unit // TODO: implement feature
+                else -> onComponentEvent(OrderHistoryEvents.ShowRatingDialog)
```

Comment:
Also the Rating Dialog doesn't show when I click on Rate us button but it works great on view all screen